src/mysql/MySQL.chpl

```diff
@@ -0,0 +1,614 @@
+module MySQL {
+    use DatabaseCommunicator.DatabaseCommunicationObjects.Implementables;
+    use DatabaseCommunicator.DatabaseCommunicationObjects.ErrorTypes;
+    use DatabaseCommunicator.QueryBuilder;
+    use CPtr;
+    use Map;
+    use SysCTypes;
+    use MySQLNative;
+    //require "mysql.h";
```

Comment:
These stray comments  should be removed or annotated to describe why we're keeping them around.